scripts/gui/microbe_editor.mjs

```diff
@@ -229,42 +238,42 @@ function setRedo(enabled){
 }
 
 //! Sends organelle selection to the Game
-function onSelectNewOrganelle(organelle){
+function onSelectNewOrganelle(element){
 
-    if(common.isInEngine()){
+    if(common.isInEngine() && !$(element.element).hasClass("DisabledButton")){
 
-        Leviathan.CallGenericEvent("MicrobeEditorOrganelleSelected", {organelle: organelle});
+        Leviathan.CallGenericEvent("MicrobeEditorOrganelleSelected",
+            {organelle: element.organelle});
 
     } else {
 
-        updateSelectedOrganelle(organelle);
+        updateSelectedOrganelle(element.organelle);
     }
 }
 
 //! Updates the GUI buttons based on selected organelle
 function updateSelectedOrganelle(organelle){
 
-    // Remove the selected text from existing ones
+    // Make all buttons unselected except the one that is now selected
```

Comment:
I *think* you broke the logic here. I don't think it properly clears the properties anymore.

Reply:
The only bug now i have if in the case there is nucleus i select for example mitochondria, then i undo nucleus mitochondria it's shown as selected but you can't put it, if you just select another one enabled then you can't select again mitochondria.

Reply:
I think the previous logic was better, so you should revert the changes to this function and put the deselection checks to the actual click callback, like I said previously that I strongly believe that this method *must* be made to take in a string and not a DOM object.

Reply:
The comment on this line is incorrect after your changes. You need to adjust the comments in this function as you changed the logic.